### PR TITLE
Update roastlist.go

### PR DESCRIPTION
**Pull request by WeepingHoney**
115 more roasts.

### CODE REVIEW

File: stdcommands/roast/roastlist.go [hunk 1]
```diff
@@ -161,4 +161,119 @@ var roasts = []string{
 	"You're the reason the gene pool needs a lifeguard.",
 	"I don't know what your problem is, but I'm guessing it's hard to pronounce.",
 	"Talking to you is like stepping on a leaf in autumn and hearing no crunch- disappointment.",
+        "You do realize we're just tolerating you, right?",
+        "Your birth certificate should be rewritten as a letter of apology.",
+        "If I were a dog and you were a flower, I'd lift my leg up and give you a shower.",
+        "You just might be why the middle finger was invented in the first place.",
+        "Your so annoying you could make a happy meal cry.",
```
Comment (ashishjh-bst) [inline stdcommands/roast/roastlist.go:168]: You're

File: stdcommands/roast/roastlist.go [hunk 2]
```diff
@@ -161,4 +161,119 @@ var roasts = []string{
...
+        "You're as usefull as a parachue during a bungie jump.",
+        "You're as much of a bright spark as a black hole.",
+        "Even a tortoise on tranquillisers can think faster then you.",
+        "You're as wanted as discord community guideline updates.",
+        "You're about as interesting as an end user licence agreement.",
+        "I feel sorry for your chair.",
+        "Expired medicine is more beneficial then your existence.",
+        "A cracked mug is less fragile then your ego",
+        "You're as scary as a kitten in a pet shop, and as needy as one too.",
+        "Your about as useful as a square wheel.",
```
Comment (ashishjh-bst) [inline stdcommands/roast/roastlist.go:189]: you're

File: stdcommands/roast/roastlist.go [hunk 3]
```diff
@@ -161,4 +161,119 @@ var roasts = []string{
...
+        "We don't have to ask if your mother drank during your pregnacy, that much is clear.",
+        "You explain every joke you make don't you?",
+        "Your birth certificate is an apology letter and compensation from the condom factory.",
+        "You look like Steven Hawking talked.",
+        "You look how wet concrete feels to walk on.",
+        "Bees choose not to sting you, you're not worth it.",
+        "You're about as menacing as a jelly-bean sized baby kangaroo.",
+        "If anyone requires an off button, it's you.",
+        "You haven't changed since the last time I saw you. You really should.",
```
Comment (ashishjh-bst) [inline stdcommands/roast/roastlist.go:198]: You're

File: stdcommands/roast/roastlist.go [hunk 4]
```diff
@@ -8,7 +8,7 @@ func randomRoast() string {
 	return roasts[rand.Intn(len(roasts))]
 }
 
-//Roasts copied from roast-bot
+//Roasts copied from roast-bot and added to by WeepingHoney
```
Comment (ashishjh-bst) [inline stdcommands/roast/roastlist.go:11]: I don't think **you're** the original creator of these roasts, I have read them all before.

**Review by ashishjh-bst — CHANGES_REQUESTED** [review]

File: stdcommands/roast/roastlist.go (hunk 3 above)
Comment (savage4618) [reply stdcommands/roast/roastlist.go:198]: I'm not seeing this one

Comment (ashishjh-bst) [reply stdcommands/roast/roastlist.go:198]: my bad, I was reviewing on phone and `you really should` as `you really stupid`

File: stdcommands/roast/roastlist.go (hunk 4 above)
Comment (mrbentarikau) [reply stdcommands/roast/roastlist.go:11]: no need for such self-inserts into the codebase, "added by" is already in repository history.

Comment (WeepingHoney) [reply stdcommands/roast/roastlist.go:11]: idk what I'm doing, I just saw a line that wasn't going to be accurate anymore and updated it. Do you have a better suggestion as leaving it as it was would be inaccurate?

Comment (galen8183) [reply stdcommands/roast/roastlist.go:11]: Perhaps add where you sourced these roasts.

**Pull request closed without merging**